gemma/configs.h

```diff
@@ -127,6 +127,10 @@ struct LayerConfig {
   size_t conv1d_width = 0;
   bool ff_biases = false;
   bool softmax_attn_output_biases = false;
+  bool self_extend = false;
+  size_t ngb_size = 0;
```

Comment:
Is this n-gram block? Maybe expand to block_size for more clarity? We can also move these three new fields into a section (just newline before them) with a `// Self-extension` comment.

Reply:
Sorry to be unclear, I was suggesting considering renaming this to ngram_block_size.
And it would be good to add a newline plus "// Self-extension" comment for visual separation from the other fields in this struct.

Reply:
Oh, `ngb` is short for `neighbour`, i see the point of confusion now.

Reply:
Ah :) Generally it's good to write out words.

Reply:
understood, i'll make the change.